package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "math-magicians",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "eslint-config-next": "13.2.4",
+    "next": "13.2.4",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
```

Comment:
Well done on this app so far 💯  but as the requirement says, the app should be created with `Create React App` as it is proven to be the best way to start [building a new single-page application](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) in React. 🏗️